lab_3/3.1/vol1.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    type: local
+  name: task-pv-volume
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 200Mi
+  hostPath:
+    path: /tmp/data
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
Where does this version come from?

Note to self - supplied version:

```yml
apiVersion: v1
kind: PersistentVolume
metadata:
  labels:
    type: local
  name: task-pv-volume
spec:
  accessModes:
  - ReadWriteOnce
  capacity:
    storage: 200Mi
  hostPath:
    path: /tmp/data
  persistentVolumeReclaimPolicy: Retain
```

Reply:
I grabbed it from the PDF - I forgot that they include some of the source files in the solutions pack.